### PR TITLE
lx-music-desktop: new, 2.8.0

### PR DESCRIPTION
**Pull request by xunpod**
Topic Description
-----------------

- lx-music-desktop: new, 2.8.0

Package(s) Affected
-------------------

- lx-music-desktop: 2.8.0

Security Update?
----------------

No

Build Order
-----------

```
#buildit lx-music-desktop
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`

### CODE REVIEW

**Comment by MingcongBai** [comment]
图片

I don't seem to be able to play anything here.

File: app-multimedia/lx-music-desktop/autobuild/beyond [hunk 1]
```diff
@@ -0,0 +1,8 @@
+abinfo "Installing icons ..."
+for size in 16x16 32x32 48x48 64x64 128x128 256x256 512x512 ; do
+	install -Dvm644 "$SRCDIR"/resources/icons/$size.png \
+		"$PKGDIR"/usr/share/icons/hicolor/$size/apps/lx-music-desktop.png
+done
+mkdir -pv "$PKGDIR"/usr/share/lx-music-desktop/
```
Comment (MingcongBai) [inline app-multimedia/lx-music-desktop/autobuild/beyond]: Use `install -Dvm644` here.

File: app-multimedia/lx-music-desktop/autobuild/defines [hunk 2]
```diff
@@ -0,0 +1,9 @@
+PKGNAME=lx-music-desktop
+PKGDES="A music searching assistant and player based on Electon"
+PKGSEC="sound"
+PKGDEP="alsa-lib desktop-file-utils flac harfbuzz hicolor-icon-theme icu libpng \
+        nss opus snappy speech-dispatcher perl-file-basedir at-spi2-core \
+        cups gtk-3 ca-certs cairo x11-lib libxcb curl libdrm mesa expat \
+        wget xdg-utils osdlyrics"
+BUILDDEP="nodejs"
+PKGPROV="lx-music"
```
Comment (MingcongBai) [inline app-multimedia/lx-music-desktop/autobuild/defines]: Please mark FAIL_ARCH.

File: app-multimedia/lx-music-desktop/autobuild/overrides/usr/share/applications/lx-music-desktop.desktop [hunk 3]
```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=LX Music
```
Comment (MingcongBai) [inline app-multimedia/lx-music-desktop/autobuild/overrides/usr/share/applications/lx-music-desktop.desktop:2]: Localise this.

File: app-multimedia/lx-music-desktop/autobuild/defines [hunk 4]
```diff
@@ -0,0 +1,9 @@
+PKGNAME=lx-music-desktop
+PKGDES="A music searching assistant and player based on Electon"
```
Comment (MingcongBai) [inline app-multimedia/lx-music-desktop/autobuild/defines]: An Internet music streaming client and collection manager

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Comment by xunpod** [comment]
> 图片
> 
> I don't seem to be able to play anything here.

It's a normal phenomenon. Due to some reasons, a custom source is required for the first use.

**Comment by MingcongBai** [comment]
Let's not introduce this package until the upstream figures out a way to make this software work out of the box.

**Pull request closed without merging**